src/makeRobotAccountant.js

```diff
@@ -4,9 +4,17 @@
  *
  * @return {function}
  */
-
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return first => {
+    count++;
+
+    return second => count > 3
+    && count % 2 === 0
+      ? 'Bzzz... Error!'
+      : first + second;
```

Comment:
```suggestion
    return second => {
      return count > 3 && count % 2 === 0
      ? 'Bzzz... Error!'
      : first + second;
    }
```
make code more readable